.github/workflows/.github/workflows/geometry-config-generator.yml

```diff
@@ -0,0 +1,89 @@
+name: Geometry config generator
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_branch:
+        description: "Target branch to generate geometry config against"
+        type: string
+        required: true
+        default: "v1.4.0"
+  push:
+    branches:
+      - add-geometry-config-generator-workflow
+
```

Comment:
Please add permission section as we discussed

Reply:
added